code/API_definitions/location-retrieval.yaml

```diff
@@ -407,16 +435,21 @@ components:
           schema:
             $ref: '#/components/schemas/ErrorInfo'
           examples:
-            notFound:
-              value:
-                status: 404
-                code: NOT_FOUND
-                message: 'The specified resource is not found'
-            deviceNotFound:
+            LOCATION_RETRIEVAL_404_UNABLE_TO_LOCATE_DEVICE:
+              summary: The location server is not able to locate the mobile

```

Comment:
Maybe better "device" instead of "mobile"

Reply:
Yes - fixed.